src/dashboard/src/pages/Jobs/index.tsx

```diff
@@ -99,6 +119,13 @@ const Jobs: FunctionComponent = () => {
     <Container fixed maxWidth="xl">
       <Toolbar disableGutters>
         <FormControl fullWidth>
+          { amlUrl != '' ? 
```

Comment:
I got what you want, just put a `useFetch` with full url and make the `clusterId` as its dependency. and then use a `useMemo` to map the cluster config to the actual `amlUrl`